app/services/scene-collections/scene-collections.ts

```diff
@@ -232,7 +231,7 @@ export class SceneCollectionsService extends Service implements ISceneCollection
     const removingActiveCollection = id === this.activeCollection.id;
 
     if (removingActiveCollection) {
-      this.appService.runInLoadingMode(async () => {
+      await this.appService.runInLoadingMode(async () => {
```

Comment:
Got race-condition issues when tried to remove several scene-collections in a row